worlds/noita/__init__.py

```diff
@@ -1,7 +1,8 @@
-from BaseClasses import Tutorial
+from BaseClasses import Tutorial, Item
 from worlds.AutoWorld import World, WebWorld
 from . import Options, Items, Locations, Regions, Rules, Events
 from .Options import noita_options
+from .Items import NoitaItem, item_table
```

Comment:
I'd prefer we use the fully qualified `Items.item_table` in the code, otherwise it's not clear where it comes from, but the change below removes it anyway.